.gitignore

```diff
@@ -21,3 +21,6 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+# :wq!
```

Comment:
errr... 😕

Reply:
It's so people know they are `vi` settings :)